src/Mod/Part/App/TopoShape.h

```diff
@@ -925,6 +925,9 @@ class PartExport TopoShape: public Data::ComplexGeoData
     void mapSubElement(const TopoShape &other,const char *op=nullptr, bool forceHasher=false);
     void mapSubElement(const std::vector<TopoShape> &shapes, const char *op=nullptr);
     void mapSubElementsTo(std::vector<TopoShape>& shapes, const char* op = nullptr) const;
+    void flushElementMap() const;
+    virtual Data::ElementMapPtr resetElementMap(
+            Data::ElementMapPtr elementMap=Data::ElementMapPtr());

```

Comment:
Switching from virtual to override on these seems appropriate.   The 'overridden' spellcheck below also makes sense

Reply:
Fine, I'll change this method from virtual to override and update the comment below.